frontend/src/app/shared/order-button-group/order-button-group.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, EventEmitter, Input, Output } from "@angular/core";
+import { actionIcons } from "@shared/icons";
+
+export type OrderChange = "up" | "down";
+
+@Component({
+    selector: "lc-order-button-group",
+    templateUrl: "./order-button-group.component.html",
+    styleUrls: ["./order-button-group.component.scss"],
+})
+export class OrderButtonGroupComponent {
+    @Input() entityName: string | null = null;
+    @Output() changeOrder = new EventEmitter<OrderChange>();
```

Comment:
You could also just emit `1` or `-1` here, which makes handling the event easier. That's a matter of personal preference, though.